README.md

```diff
@@ -1 +1,119 @@
+        'code': opensearch_fields.KeywordField(),
+        'name': opensearch_fields.KeywordField(),
+    })
+    individual = opensearch_fields.ObjectField(properties={
+        'first_name': opensearch_fields.KeywordField(),
+        'last_name': opensearch_fields.KeywordField(),
+        'dob': opensearch_fields.DateField(),
+    })
+    status = opensearch_fields.KeywordField(fields={
+        'status_key': opensearch_fields.KeywordField()}
+    )
+
+    class Index:
+        name = 'beneficiary'  # Name of the Opensearch index
+        settings = {
+            'number_of_shards': 1,
+            'number_of_replicas': 0
+        }
+        auto_refresh = True # automatically send updates and new objects to openSearch
+
+    class Django:
+        model = Beneficiary
+        related_models = [BenefitPlan, Individual]
+        fields = [
+            'id',
+        ]
+        queryset_pagination = 5000
+```
+* In the Django class, you can use the `fields` property to include all 
+the fields that are present in your Django ORM model.
+* If you wish to include related fields (foreign keys), make sure to specify them 
+in the `related_models` property of your Django class. Add all the foreign key fields 
+that you want to include.
+```python 
+    class Django:
+        model = Beneficiary
+        related_models = [BenefitPlan, Individual]
+        fields = [
+            'id',
+        ]
+        queryset_pagination = 5000
+``` 
+* If you want to customize the default behavior of field creation, declare the fields you wish to modify 
+using the appropriate type. For foreign keys, use the `ObjectField` type.
+```python
+  @registry.register_document
+  class BeneficiaryDocument(Document):
+      benefit_plan = opensearch_fields.ObjectField(properties={
+          'code': opensearch_fields.KeywordField(),
+          'name': opensearch_fields.KeywordField(),
+      })
+      individual = opensearch_fields.ObjectField(properties={
+          'first_name': opensearch_fields.KeywordField(),
+          'last_name': opensearch_fields.KeywordField(),
+          'dob': opensearch_fields.DateField(),
+      })
+      status = opensearch_fields.KeywordField(fields={
+          'status_key': opensearch_fields.KeywordField()}
+      )
+```
+* Note that `TextField` types are typically converted to non-aggregable fields in OpenSearch. 
+If you require aggregable fields, use the `KeywordField` type.
+* By default, the `auto_refresh` property in the Index class should enable data to be 
+automatically transferred on every CRUD operation managed by Django ORM. Ensure that 
+this property is properly configured in your project.
+
+
+## How to initialize data after deployment
+* If you have initialized the application but still have some data to be transferred, you can effortlessly 
+achieve this by using the commands available in the business module: `python manage.py add_<model_name>_data_to_opensearch`. 
+This command loads existing data into OpenSearch.
```

Comment:
I think we could put a reference to opensearch setup in the assembly's module readme

Reply:
ok